### DISCUSSION TITLE
How to change route from /gallery to /albums?

### DISCUSSION THREAD

**Discussion by wizgod** [bpatrik/pigallery2#380] [Q&A]
Greetings Programs!

My domain tld is .gallery so having the url show as domain.gallery/gallery looks funny.

How can I change the route so that the url shows as domain.gallery/albums instead of /gallery?

Thanks,

Wg

**Comment by bpatrik** [comment]
you cannot really do that. Especially as album(s) is also a valid route for the app. 

You can change the code tho, in both the frontend and the backend.

**Reply by wizgod** [reply]
Hi @bpatrik,

Is there a central location for the routes code in the frontend and backend or would it be more troublesome to try and change the routes and probably should just leave it?

Thanks,

Wg

**Comment by bpatrik** [comment]
not 100% sure, but probably you need to change these:
https://github.com/bpatrik/pigallery2/blob/22954550bfd473d85206d62aca50e47853b20862/src/frontend/app/app.routing.ts#L23
https://github.com/bpatrik/pigallery2/blob/22954550bfd473d85206d62aca50e47853b20862/src/backend/routes/PublicRouter.ts#L117

**Reply by wizgod** [reply]
Thanks @bpatrik, I will start there and see how it goes.

Wg
